css/main.css

```diff
@@ -0,0 +1,128 @@
+@import "./reset.css";
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+  position: relative;
+  cursor: pointer;
+  transition-property: transform background-color;
+  transition: 500ms;
+}
+.calendar__day::before {
+  font-family: inherit;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.calendar__day:nth-child(1)::before {
+  content: "1";
+}
+.calendar__day:nth-child(2)::before {
+  content: "2";
+}
+.calendar__day:nth-child(3)::before {
+  content: "3";
+}
+.calendar__day:nth-child(4)::before {
+  content: "4";
+}
+.calendar__day:nth-child(5)::before {
+  content: "5";
+}
+.calendar__day:nth-child(6)::before {
+  content: "6";
+}
+.calendar__day:nth-child(7)::before {
+  content: "7";
+}
+.calendar__day:nth-child(8)::before {
+  content: "8";
+}
```

Comment:
use `for` to set content

@for $day from 1 through $month-length {
      &:nth-child(#{$day})::before {
        content: "#{$day}";
      }
    }

Reply:
So, i see you have already used a loop for creating date, it means that you don't need these any more

.....
`.calendar__day:nth-child(7)::before {
  content: "7";
}
.calendar__day:nth-child(8)::before {
  content: "8";
}`
.....